977.squares-of-a-sorted-array.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/squares-of-a-sorted-array/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number[]}
+ */
+var sortedSquares = function(nums) {
+    
+    let result = [];
```

Comment:
```suggestion
    const result = [];
```
Const is better here